src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -287,17 +287,23 @@ internal partial class CodeFixService : ForegroundThreadAffinitizedObject, ICode
                 // If the fix provider supports fix all occurrences, then get the corresponding FixAllProviderInfo and fix all context.
                 var fixAllProviderInfo = extensionManager.PerformFunction(fixer, () => ImmutableInterlocked.GetOrAdd(ref _fixAllProviderMap, fixer, FixAllProviderInfo.Create), defaultValue: null);
 
-                FixAllCodeActionContext fixAllContext = null;
+                FixAllProvider fixAllProvider = null;
+                FixAllContext fixAllContext = null;
+                IEnumerable<FixAllScope> supportedScopes = null;
                 if (fixAllProviderInfo != null)
                 {
                     var codeFixProvider = (fixer as CodeFixProvider) ?? new WrapperCodeFixProvider((ISuppressionFixProvider)fixer, diagnostics.Select(d => d.Id));
-                    fixAllContext = FixAllCodeActionContext.Create(
+                    fixAllProvider = fixAllProviderInfo.FixAllProvider;
+                    fixAllContext = FixAllContext.Create(
```

Comment:
As we discussed offline, you can probably change this later to create a FixAllContextBuilder, which also stores the supported scopes.